documentation/WIDGET.md

```diff
@@ -12,6 +12,17 @@
 </resources>
 ```
 
+#### If your app targets Android 14 (API level 34) or higher you must specify a foreground service type for all long-running workers.
```

Comment:
I think this change (22->34) was made when we created background uploads for the widget. Am I right?

Reply:
This change (22 -> 34) was made when we raised the targetSdk version to 34 (the current version). Previously, it was 30, as the targetSdk version was 30 before.